pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "kcc3"
+version = "0.1.0"
+description = ""
+authors = ["Mateusz Maćkowski <m4tx@m4tx.pl>"]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
```

Comment:
```suggestion
python = "^3.12"
```

Reply:
Let's change it after Arch Linux upgrades to 3.12.